default.nix

```diff
@@ -171,8 +175,9 @@ in rec {
       inherit name;
 
       configurePhase = ''
-        cp -r ${nodeModules}/node_modules ./node_modules
+        cp --reflink=auto -r ${nodeModules}/node_modules ./node_modules
         chmod -R u+w ./node_modules
+        patchShebangs .
```

Comment:
`./node_modules` is already patched, if you want to patch the sources do it before copying `node_modules` to avoid going through it again